### PR TITLE
Completed ticket 2

### PR DESCRIPTION
**Pull request by NiallSexton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,57 +1,5 @@
...
-```
-git branch -D <local branch>
-```
-
-## Husky
-
-To ensure we are not commiting broken code this project makes use of git hooks. Git hooks are scripts triggered during certain events in the git lifecycle. Husky is a popular package which allows us to set up and maintain these scripts. This project makes use a _pre-commit hook_. When we attempt to commit our work, the script defined in the `pre-commit` file will run. If any of our tests fail than the commit will be aborted.
-
-The [Husky documentation](https://typicode.github.io/husky/#/) explains how to configure Husky for your own project as well as creating your own custom hooks.\_
+2- To connect to the test database, create a new folder called '.env.test' and copy and save the following code into it 'PGDATABASE=nc_games_test;'
```
Comment (two-triangles) [inline README.md:5]: You need to add that we need to .gitIgnore these files.

File: README.md [hunk 2]
```diff
@@ -1,57 +1,5 @@
-# Northcoders House of Games API
+Upon cloning the repository, certain files will not be availabe and in order to connect to the Games Review databases, you must create the following files:
```
Comment (two-triangles) [inline README.md:1]: *available

**Comment by two-triangles** [comment]
Left some comments, have look before merging

**Review by itskatherine — COMMENTED** [review]
Nice work Niall! Maybe check the previous comment re spelling mistakes, but good to merge! 👍

**Pull request merged**